sideboard/config.py

```diff
@@ -38,36 +99,64 @@ def get_dirnames(pyname, plugin):
     package under the configured "plugins_dir" directory.
 
     These two directories are also automatically inserted into plugin config
-    files as "root" and "module_root" and are available for interpolation.  For
-    example, a plugin could have a line in their config file like
+    files as "root" and "module_root" and are available for interpolation. For
+    example, a plugin could have a line in their config file like::
```

Comment:
Wow! We've got some eagle-eyed reviewers here! I'm impressed. However, I did intentionally use two colons. In reStructuredText (which is the de facto standard for docstrings) two colons indicates the start of a [literal block](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks) (as you'd use for code or other mono-space reformatted blocks).